lib/repl.js

```diff
@@ -1195,3 +1195,4 @@ function Recoverable(err) {
   this.err = err;
 }
 inherits(Recoverable, SyntaxError);
+exports.Recoverable = Recoverable;
```

Comment:
Actually, @princejwesley, can we do that in https://github.com/nodejs/node/pull/6171. The signature will need to change later for the line parser changes so it'd be better to expose it later.